Configuration/TypoScript/Styles/Twb5/setup.typoscript

```diff
@@ -0,0 +1,34 @@
+plugin.tx_news {
+	view {
+		templateRootPaths {
+			0 = EXT:news/Resources/Private/Templates/
+			1 = EXT:news/Resources/Private/Templates/Styles/Twb5/Templates
+			2 = {$plugin.tx_news.view.twb.templateRootPath}
+		}
+
+		partialRootPaths {
+			0 = EXT:news/Resources/Private/Partials/
+			1 = EXT:news/Resources/Private/Templates/Styles/Twb5/Partials/
+			2 = {$plugin.tx_news.view.twb.partialRootPath}
+		}
+
+		layoutRootPaths {
+			0 = EXT:news/Resources/Private/Layouts/
+			1 = EXT:news/Resources/Private/Templates/Styles/Twb5/Layouts/
+			2 = {$plugin.tx_news.view.twb.layoutRootPath}
```

Comment:
Maybe I'm missing something but shouldn't these 3 constants refer to the `twb5` constants above?